src/storageDataGroup.js

```diff
@@ -112,7 +112,7 @@ function updateDataGroup(path, records) {
         return deleteDataGroup(path);
     }
     const randBytes = crypto.randomBytes(16).toString('hex');
-    const configFileName = pathLib.join(os.tmpdir(), `atg-storage.${randBytes}.conf`);
+    const configFileName = pathLib.join(os.tmpdir(), `tmp${randBytes}.conf`);
```

Comment:
Isn't there still a concern of having the file name be partially predictable? For example, could you just search for `tmp*` instead of `atg-storage*`?

Reply:
I would suspect so, but I am not finding information on the prefix being insecure. `mkstemp()` still  uses a prefix, and Python's temporary file utilities use a default prefix of `tmp`. Taking a look at a BIG-IP, I am not seeing any other temporary files with a `tmp` prefix, so it would still be pretty easy to pick out one belonging to atg-storage. Perhaps the permissions updates are enough to mitigate this concern?

Reply:
We can at least get these changes in and continue to find ways to improve security.